src/test/java/seedu/duke/task/TaskTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.duke.task;
```

Comment:
Good job creating test cases for the repeat function to ensure the bugs are fixed!